### ISSUE TITLE
v0.5.6 of SparkJS fails Simple Example Use

### ISSUE THREAD

**Issue by anthony-ngu** [spark/sparkjs#66]
In the latest update, calling Sparkjs fails.

I am getting an oauth error from the generic sample with my username and password correctly inputted.

This is the code:
```
var Spark = require('spark');

Spark.login({ username: 'email@example.com', password: 'password' }, function(err, body) {
  console.log('API call login completed on callback:', body);
});
```

This is the error being returned to me:
```[Error: Invalid URI "[object%20Object]/oauth/token"]```

I was using v0.4.2 and updated recently to find that v0.5.6 fails this simple task..

**Comment by emilyrose** [comment]
Hey @anthony-ngu, thanks for the report!

My personal apologies that this was broken for you; I was the one that broke it! The tests passed, but I should have tried things out myself. I was hastily attempting to unblock @suda in his quest, and dropped the ball on quality assurance (I was super tired :smile_cat:).

At any rate, I fixed it here: 6bd9b8f76be6c5b3880bd73913e7c804eb7bada6

Thanks again for reporting this, and I will do my best to avoid such embarrassing mistakes in the future! 

:cold_sweat: — Emily

**Comment by anthony-ngu** [comment]
:+1: I forgot to check up on this since I reverted back to a previous version, but it looks good now
